### ISSUE TITLE
Improve ease of search on website/book

### ISSUE THREAD

**Issue by madsonic** [nus-cs2103/website#24]
While it is possible to search for stuff on either page, it is assumed that the user know which section it belongs to so that it can be expanded and searched but this assumption reduces the effective of search. 

Are there requests from previous students that searching through the entire content without expansion as desirable? Maybe we could try to reduce the time needed for them/us to search for stuff

**Comment by damithc** [comment]
Yes, a search feature can be useful. It needs a lot of extra work to add such a major feature to the underlying platform, so will not happen anytime soon.
